libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/Consensus.hs

```diff
@@ -45,7 +47,7 @@ import Cardano.Ledger.Conway.Genesis (ConwayGenesis (..))
 import qualified Cardano.Ledger.Conway.PParams as ConwayPP
 import Cardano.Ledger.Conway.Translation ()
 import Cardano.Ledger.Core as Core
-import Cardano.Ledger.Crypto as CC
+import Cardano.Ledger.Crypto
```

Comment:
This `CC`  alias for `Cardano.Ledger.Crypto`  seems to be used a lot throughout  the codebase, but perhaps it's necessary in some places as opposed to here

Reply:
We used to have a clash, these two had the same name `Cardano.Ledger.Crypto.Crypto` and `Cardano.Lerdge.Core.EraCrypto`. This resulted in all those qualified as CC. I suggested a rename ` Crypto` -> `EraCrypto` #2933, so this is no longer necessary.